VictorMaria/express-dataset
# #168970269 Create event

#### What does this PR do?
Creates the functionality that allows a registered user to create an event

#### Description of Task to be completed?
Has POST http://localhost:3000/events endpoint working

#### How should this be manually tested?
1. Fetch the branch using ```git fetch origin ft-create-event-168970269```
2. Install dependences using ```npm install```
3. Start the server using ```npm run start``` or ```npm run launchapp```
4. Sign into the app and paste the token received from the response in the x-access-token field in POSTMAN header's section.
5. See the example below to fill the required fields
id: yourRepoID
name: yourRepoName
url: yourRepoURL
6. Launch a POST request on POSTMAN using http://localhost:3000/events
7. Wait for a response from the server
#### What are the relevant pivotal tracker stories?
[#168970269](https://www.pivotaltracker.com/story/show/168970269)
